### DISCUSSION TITLE
来绳网的第一天水一帖

### DISCUSSION THREAD

**Discussion by ShuiJingOfficial** [share121/inter-knot#1419] [其他]
你怎么知道
![5e13a5b11ee7f87e16060eea2bb5b2a9](https://github.com/user-attachments/assets/442f9a8f-ca40-4e9c-a45e-0a67c886f3af)
![2f7acaef4339ae3bd530d1c806b9c1a5](https://github.com/user-attachments/assets/6bbed603-da36-4dd1-9aea-a644835798a2)
![b0ba7a6270dc51d1afa28de0161ebe7c](https://github.com/user-attachments/assets/a941592c-5cea-446d-a57b-2ef8269f9597)
![f75d12de6d077611894e5c228f3e2e79](https://github.com/user-attachments/assets/4da2f0a8-b711-41fa-9271-ffc161246f53)
![cf00b974bc7e25ea0f304de6fb423a9f](https://github.com/user-attachments/assets/316bdb11-36ca-434b-aa7e-a4c56dd4d7c0)

**Comment by Kaohaaa** [comment]
逆天，@ 治安群的人把你抓走

**Comment by ZhuYuan-148** [comment]
只抓绳匠，绳斤不归治安局管（开个玩笑）
